atlas-content-modeler/tutorial/models/taxonomies/query.mdx

```diff
@@ -0,0 +1,117 @@
+---
+slug: /atlas-content-modeler/tutorial/models/taxonomies/query
+title: Atlas Content Modeler Taxonomies - Query
+description: Learn how to Query Taxonomies in Atlas Content Modeler.
+---
+
+**Note**: These tutorials assume that you have already [installed the plugin](../../../introduction.mdx).
+
+## Query WPGraphQL for a Taxonomy
+
+Learn how to query taxonomies using WPGraphQL.
+
+---
+
+### Dependencies
+
+#### WPGraphQL
+
+Please download and install WPGraphQL following these instructions: [WPGraphQL Installation - Quick Start](https://www.wpgraphql.com/docs/quick-start/).
+
+#### WPGraphQL Tax Query
+
+Please download, install, and activate **WPGraphQL Tax Query** using the [Wordpress Plugin Installation Method](https://wordpress.org/support/article/managing-plugins/#automatic-plugin-installation).
```

Comment:
Worth linking to the plugin here?

Also, I think people will have to install manually because WPGraphQL Tax Query isn't in the WordPress.org repo.

```suggestion
Please download, install, and activate [WPGraphQL Tax Query](https://github.com/wp-graphql/wp-graphql-tax-query) by [uploading the plugin zip file in the WordPress admin](https://wordpress.org/support/article/managing-plugins/#manual-upload-via-wordpress-admin).
```

Reply:
Updated